services/search.ts

```diff
@@ -161,6 +161,49 @@ export class SearchService extends Service {
     res.status(response.code).json(response)
   }
 
+  /**
+   * @description Add search query to the database
+   * @param {express.Request} req
+   * @param {express.Response} res
+   */
+  public async query(req: express.Request, res: express.Response) {
+    let response: APIresponse = {
+      ok: 1,
+      code: 200,
+    }
+    const rawQuery = req.body.search
+    const query = rawQuery.replace(/\s/g, '').toLowerCase() // removing whitespaces and lowercase the query
+
+    try {
+      response.search = {}
+
+      const checkQuery = `SELECT query FROM searchQuery WHERE query LIKE ?`
+      let checkExistingQuery = await this.database.query(checkQuery, [
+        `${query}`,
```

Comment:
This is what you meant right? @MathisBullinger 
I didn't add the % in the beginning and the end of the params like Fabio did cause it changed the search query.

for example:
`%${search.toLowerCase()}%`,

Hope it make sense

Reply:
Yes, that is probably fine. Just one thing: `` `${query}` `` is equivalent to just writing `query`.

Reply:
you are right, I will push another commit fixing it